must-gather/collection-scripts/gather_nro

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+crdoi_list=()
+crdoi_list+=$(oc get crds -o=custom-columns=NAME:.metadata.name --no-headers | grep -E "^*\.nodetopology\.openshift\.io$")
+crdoi_list+=( noderesourcetopologies.topology.node.k8s.io )
```

Comment:
we know the names of all the crds we need, and these names are expected to be stable. What's the advantage of autodetecting some and hardcoding some other? I'd just hardcoded all of them.